src/Chat/ChatInterface.php

```diff
@@ -35,4 +35,6 @@ public function setFunctions(array $functions): void;
     public function addFunction(FunctionInfo $functionInfo): void;
 
     public function setModelOption(string $option, mixed $value): void;
+
+    public function getTotalTokens(): int;
```

Comment:
I'm not sure if it is a good thing to pollute the ChatInterface with this method that is specific to OpenAI

Reply:
We already could get the number of tokens with Anthropic. I don't see how to get this value with Ollama and I'm not sure this will be available any time soon.

Reply:
Do you mean in the `QuestionAnswering` class? I think ot could be a good option.